jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -164,6 +164,11 @@
     @NotNull
     private final EndpointConnectionStatusMonitor epConnectionStatusMonitor;
 
+    /**
+     * A unique meeting ID set by the signaling server. It is exposed via ({@link #getDebugState()} for outside use.
```

Comment:
Could you add a comment about the fact that it's nullable, and what it means for it to be null vs not?